### ISSUE TITLE
Invalid arm compiler url in the ubuntu dockerfiles

### ISSUE THREAD

**Issue by lpy4105** [Mbed-TLS/mbedtls-test#77]
Due to some change the https://developer.arm.com/, the url of ARM Compiler in ubuntu dockerfiles are invalid. Thus, it cause errors when building the docker images.
```
Step 9/45 : RUN wget -q https://armkeil.blob.core.windows.net/developer/Files/downloads/compiler/DS500-PA-00003-r5p0-22rel0.tgz &&     tar -zxf DS500-PA-00003-r5p0-22rel0.tgz &&     ./Installer/setup.sh --i-agree-to-the-contained-eula --no-interactive -d /usr/local/ARM_Compiler_5.06u3 --quiet &&     rm -rf DS500-PA-00003-r5p0-22rel0.tgz releasenotes.html Installer/
 ---> Running in cab693a72fb9
The command '/bin/sh -c wget -q https://armkeil.blob.core.windows.net/developer/Files/downloads/compiler/DS500-PA-00003-r5p0-22rel0.tgz &&     tar -zxf DS500-PA-00003-r5p0-22rel0.tgz &&     ./Installer/setup.sh --i-agree-to-the-contained-eula --no-interactive -d /usr/local/ARM_Compiler_5.06u3 --quiet &&     rm -rf DS500-PA-00003-r5p0-22rel0.tgz releasenotes.html Installer/' returned a non-zero code: 8
```

**Comment by yuhaoth** [comment]
I have push all package in https://hub.docker.com/repository/docker/yuhaoth/sources . To use that put below line in dockerfile.

```dockerfile
copy --from=yuhaoth/sources pre-builts/DS500-PA-00003-r5p0-22rel0.tgz /opt
```

**Comment by gilles-peskine-arm** [comment]
We need to find a permanent hosting place, preferably controlled by the team so that we don't depend on another organization or on a specific person.

**Comment by gilles-peskine-arm** [comment]
Resolved with a workaround in #125. Resolved more cleanly in #180.
